platform/kube/controller.go

```diff
@@ -363,6 +363,26 @@ const (
 // "spiffe://cluster.local/ns/foo/sa/bar".
 func (c *Controller) GetIstioServiceAccounts(hostname string, ports []string) []string {
 	saSet := make(map[string]bool)
+
+	// Get the service accounts allowed for the service running on VMs, specified by the Service
```

Comment:
Can you be more specific about the standard k8s syntax?
Having multiple service accounts in VMs is a feature to support multi-tenant per VM. There will be multiple workloads on a VM, and each workload can have a service account identity. This is not a feature for V0.2 though.